CMP/Setup/SetupCMP/Helpers/Constants.cs

```diff
@@ -274,19 +274,17 @@ public static string WapDBName
         public static bool DBOnRemoteServer
         {
             get
-            {
-                int? returnValue = RegistryUtils.ReadRegistryValue(SetupConstants.SqlSettingsRegKey, SetupConstants.OnRemoteRegistryValueName, 0, false) as int?;
-
-                return returnValue.HasValue ? (returnValue.Value == 1) : false;
+            {
+                string returnValue = Convert.ToString(RegistryUtils.ReadRegistryValue(SetupConstants.SqlSettingsRegKey, SetupConstants.OnRemoteRegistryValueName, 0, false));
+                return (returnValue == "1") ? true : false;
```

Comment:
This part the (return true or false) can be omitted, if you'd like. You can just return the conditional itself, which will be evaluated and will return directly your true or your false.

Very minor thing, though.